django/utils/http.py

```diff
@@ -293,12 +295,21 @@ def is_safe_url(url, host=None):
             url = force_text(url)
         except UnicodeDecodeError:
             return False
+    if allowed_hosts is None:
+        allowed_hosts = set()
+    if host:
+        warnings.warn(
```

Comment:
Hah, gotcha. Didn't see that :-) I think I would go with `host` and `hosts` as argument names and drop the deprecation warning.